common/test/acceptance/tests/helpers.py

```diff
@@ -350,6 +350,48 @@ def is_404_page(browser):
     return 'Page not found (404)' in browser.find_element_by_tag_name('h1').text
 
 
+def get_url_with_changed_domain(url):
+    """
+    Replaces .org with .io in the url
+
+    Arguments:
+        url (str): The url to perform replace operation on.
+
+    Returns:
+        str: The updated url
+    """
+    return url.replace('.org/', '.io/')
+
+
+def assert_links(test, expected_link, actual_link):
+    """
+    Assert that 'href' and text are the same as expected.
```

Comment:
Can you update the docstring with little bit more explanation. I am unable to get what are `href` and `text` here?